src/uwuw/uwuw.cpp

```diff
@@ -74,24 +74,10 @@ bool UWUW::check_file_exists(const std::string& filename) {
 
 // loads all materials into map
 
-std::map<std::string, pyne::Material> UWUW::load_pyne_materials(std::string filename, std::string datapath) {
-  std::map<std::string, pyne::Material> library; // material library
-
-  const char* data_path = datapath.c_str();
-
-  if (!hdf5_path_exists(filename, data_path))
-    return library;
-
-  num_materials = get_length_of_table(filename, datapath);
-
-  for (int i = 0 ; i < num_materials ; i++) {
```

Comment:
this is the part of the code that gave unique material numbers that I was thinking of, appoligies - now that its done on the pairs thats probably better.

Reply:
no worries, this lead to an good improvement, glad you bring that up :)